discovery-provider/plugins/pedalboard/apps/app-template/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM node:18-alpine AS base
```

Comment:
this dockerfile actually isn't used, need to remove it. i'm keeping it for now as a template for the main dockerfile